src/app/Providers/components/VMwareProviderHostsTable/VMwareProviderHostsTable.tsx

```diff
@@ -25,11 +25,9 @@ const VMwareProviderHostsTable: React.FunctionComponent<IVMwareProviderHostsTabl
 
   const getSortValues = (host: IHost) => {
     const { name, network, bandwidth, mtu } = host;
-    // First cell is the generated checkbox from using onSelect.
-    return ['', name, 'network', 'bandwidth', 'mtu'];
-    //TODO update when api data is available
-    // return ['', name, formatHostNetwork(network), bandwidth, mtu];
+    return ['', name, formatHostNetwork(network), bandwidth, mtu];
```

Comment:
Looking good so far.. a few issues are showing up for me in remote mode. Are you using VScode? The ts compiler is complaining about a few things on this page 

![image](https://user-images.githubusercontent.com/11218376/95359586-5770e600-0898-11eb-9bde-3723efb30c00.png)

Reply:
There was an existing issue, I've removed unused attribute of `UsePaginationState`.
I can't see anything else and `yarn ci` doesn't report any error on that component/page.
I do use VSCode and I've seen it lagging sometimes forcing me to reload editor.

Reply:
Hm, I can't reproduce those errors you were seeing @ibolton336

Reply:
@mturley I think my editor was using a newer version of ts. 4.0.9. Just changed it to use the workspace version.

Reply:
@ibolton Re: handling the case where a host has an undefined network, maybe that's the right solution but I'm not sure. Depending on how we get the real network data from the API, wherever it actually is it might always be defined? I guess we do need to think about a case where there is no network selected for a host.

Reply:
@mturley Ah yeah the undefined check will probably at the component render function level so probably no need to check for undefined here. +1 for setting the network cell back to a 'network' string in remote mode for now until #85 is fixed.